marconi-mamba/README.md

```diff
@@ -1,78 +1,174 @@
-marconi-mamba
---
 
-This is an initial draft for the Marconi Mamba README file. Going forward, we will need to discuss and refine this document and/or the implementation so that they match.
+# Marconi-Mamba
 
-## Description
-marconi-mamba is a [JSON-RPC](http://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0) HTTP server on top of the [marconi](../marconi/README.md)
+Marconi-Mamba is a lightweight, customizable chain follower application and library for DApp developers who need to index and query the Cardano blockchain. Marconi-Mamba provides a way for DApps to have access to data that has already been stored on-chain by indexing portions of the blockchain.
 
-## Playing with the JSON-RPC
-There is an example client/server JSON-RPC to experiment with the JSON-RPC clien/server. Here are the steps to build and execute the examples:
+## Purpose
+
+The purpose of Marconi-Mamba is to make the core Marconi APIs available to non-Haskell applications.
+
+## Interface
+
+The interface for Marconi-Mamba uses [JSON-RPC](http://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0) (JSON remote-procedure calls) over HTTP built on top of [Marconi](../marconi/README.md).
 
-#### Build the project
 
-``` sh
-cabal build all
-cd marconi-mamba
-cabal build
 ```
+             Running on a single machine                    Internet
++----------------------------------------------------+
+|                                                    |                  +-----------+
+|   +----------------+                +-----------+  |                  |           |
+|   |                | node-to-client |           |  |     JSON-RPC     |  marconi  |
+|   |  cardano-node  +----------------+  marconi  +--+------------------+  client   |
+|   |                |      IPC       |           |  |       HTTP       |           |
+|   +----------------+                +----+------+  |                  +-----------+
+|                                          |         |
+|                                          |         |
+|                                      +---+----+    |
+|                                      | SQLite |    |
+|                                      +--------+    |
+|                                                    |
++----------------------------=-----------------------+
+```
+
+## Requirements
+* Local instance of cardano-node, version 1.35.3
```

Comment:
Not sure putting the version here is wise, as it is bound to change. I has already changed to 1.35.4.  How about,
compatible version of [cardano-node](https://github.com/input-output-hk/plutus-apps/blob/main/cabal.project#L246)

PS. I am pretty sure this was my error as I think I put the `version` there

Reply:
Yes, the version number was already there. I'll replace with what you're recommending including the link.